### ISSUE TITLE
Failing Example

### ISSUE THREAD

**Issue by redijedi** [smoketurner/dropwizard-zipkin#15]
Tried the example. It is failing. Please advise:

```
$ java -jar target/zipkin-example-1.0.3-3-SNAPSHOT.jar server
Exception in thread "main" java.lang.IllegalArgumentException: No serializer found for class com.github.kristofa.brave.Sampler$1 and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS) ) (through reference chain: com.example.helloworld.HelloWorldConfiguration["zipkin"]->com.smoketurner.dropwizard.zipkin.LoggingZipkinFactory["sampler"])
	at com.fasterxml.jackson.databind.ObjectMapper.valueToTree(ObjectMapper.java:2591)
	at io.dropwizard.configuration.YamlConfigurationFactory.build(YamlConfigurationFactory.java:105)
	at io.dropwizard.cli.ConfiguredCommand.parseConfiguration(ConfiguredCommand.java:126)
	at io.dropwizard.cli.ConfiguredCommand.run(ConfiguredCommand.java:72)
	at io.dropwizard.cli.Cli.run(Cli.java:75)
	at io.dropwizard.Application.run(Application.java:79)
	at com.example.helloworld.HelloWorldApplication.main(HelloWorldApplication.java:32)
Caused by: com.fasterxml.jackson.databind.JsonMappingException: No serializer found for class com.github.kristofa.brave.Sampler$1 and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS) ) (through reference chain: com.example.helloworld.HelloWorldConfiguration["zipkin"]->com.smoketurner.dropwizard.zipkin.LoggingZipkinFactory["sampler"])
	at com.fasterxml.jackson.databind.JsonMappingException.from(JsonMappingException.java:270)
	at com.fasterxml.jackson.databind.ser.impl.UnknownSerializer.failForEmpty(UnknownSerializer.java:68)
	at com.fasterxml.jackson.databind.ser.impl.UnknownSerializer.serialize(UnknownSerializer.java:32)
	at com.fasterxml.jackson.module.afterburner.ser.ObjectMethodPropertyWriter.serializeAsField(ObjectMethodPropertyWriter.java:87)
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:678)
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeWithType(BeanSerializerBase.java:569)
	at com.fasterxml.jackson.module.afterburner.ser.ObjectFieldPropertyWriter.serializeAsField(ObjectFieldPropertyWriter.java:89)
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:678)
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:157)
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:130)
	at com.fasterxml.jackson.databind.ObjectMapper.writeValue(ObjectMapper.java:2444)
	at com.fasterxml.jackson.databind.ObjectMapper.valueToTree(ObjectMapper.java:2586)

```

Disabling FAIL_ON_EMPTY_BEANS has no affect.

`environment.getObjectMapper().disable(SerializationFeature.FAIL_ON_EMPTY_BEANS);`

**Comment by jplock** [comment]
I think I know what's causing this.  I'll publish up a new build tonight.

**Comment by jplock** [comment]
This should be fixed in `1.0.5-2` which I just pushed to Maven Central
